cg/cli/balsamic.py

```diff
@@ -25,9 +26,16 @@ def balsamic(context):
 @balsamic.command()
 @click.option('-d', '--dry', is_flag=True, help='print config to console')
 @click.option('--target-bed', required=False, help='Optional')
+@click.option('--singularity', default="/home/proj/production/cancer/BALSAMIC/BALSAMIC/containers"
```

Comment:
Can we move this default to the config instead?

Reply:
We will. We should also add the reference and environment to the cg config. But separate PR to be done for that.